spec/requests/citizens/legal_aid_applications_controller_spec.rb

```diff
@@ -38,6 +38,25 @@
       expect(session["page_history_id"]).not_to be_nil
     end
 
+    context "when the applicant has reached the check your answers page" do
+      let(:legal_aid_application) do
+        create(
+          :application,
+          :with_applicant,
+          :with_non_passported_state_machine,
+          :checking_citizen_answers,
+          completed_at:,
+          applicant: build(:applicant, first_name: "Test", last_name: "Applicant"),
+          provider: build(:provider, firm: build(:firm, name: "Test Firm")),
```

Comment:
Do we need to specify any of this data explicitly? 

I don't see any relationship between the test data and the test itself, but perhaps I'm missing something 🤔